net-misc/cadaver/cadaver-0.23.3.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A command line WebDAV client"
+DESCRIPTION="It supports file upload, download, on-screen display, namespace ope
+rations (move/copy), collection creation and deletion, and locking operations."
+HOMEPAGE="http://www.webdav.org/cadaver"
```

Comment:
```diff
-DESCRIPTION="It supports file upload, download, on-screen display, namespace ope
-rations (move/copy), collection creation and deletion, and locking operations."
+DESCRIPTION="It supports file upload, download, on-screen display, namespace \
+operations (move/copy), collection creation and deletion, and locking \
+operations."
-HOMEPAGE="http://www.webdav.org/cadaver"
+HOMEPAGE="http://www.webdav.org/cadaver/"
```
Regarding the change for `HOMEPAGE`, the reason for it is that when you run `curl --head http://www.webdav.org/cadaver` you get a `HTTP/1.1 301 Moved Permanently` with `Location: http://www.webdav.org/cadaver/` so adding the trailing slash is recommended. Thanks!